readme.md

```diff
@@ -17,11 +17,15 @@ $ npm install --save ansi-styles
 ## Usage
 
 ```js
-const ansi = require('ansi-styles');
+const style = require('ansi-styles');
 
-console.log(ansi.green.open + 'Hello world!' + ansi.green.close);
-```
+console.log(style.green.open + 'Hello world!' + ansi.green.close);
 
+// color conversion between 16/256/16 million colors
```

Comment:
`16/256/16 million` looks wrong. It makes it read as it's either 16 million or 256 million or 16 million.